subprojects/core/src/main/java/net/covers1624/wt/WorkspaceTool.java

```diff
@@ -305,6 +305,9 @@ private void run(String[] args) throws Exception {
             }
         });
 
+        // add run directories as excluded paths for every module
+        context.workspaceScript.getWorkspace().getRunConfigContainer().getRunConfigs().values().stream().map(RunConfig::getRunDir).forEach(path -> allModules.forEach(module -> module.addExclude(path)));
```

Comment:
Put some new lines in this for readability.
```suggestion
        context.workspaceScript.getWorkspace().getRunConfigContainer().getRunConfigs().values().stream()
                .map(RunConfig::getRunDir)
                .forEach(path -> allModules.forEach(module -> module.addExclude(path)));
```